presto-product-tests/conf/docker/singlenode-kafka/docker-compose.yml

```diff
@@ -1,14 +1,22 @@
 version: '2'
 services:
+  zookeeper:
```

Comment:
why do we need zookkeeper now? Could we just set version of `spotify/kafka`

Reply:
It was used previously as well. But it was embedded.

Reply:
See https://github.com/prestosql/presto/pull/2327/files#diff-6b0bd2529e837ebd11eb9971552d39afR133